content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,187 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires and more!
+descriptionLong: |
+  Monthly updates are here! Read all about what our community is doing
+  with DVC, our CEO's interviews and talks, new team members and more!
```

Comment:
Didn't expand on use cases or videos here.

Reply:
BTW I'm not sure where short vs long descriptions display. Is the short one for SEO? If so it should have a max length, probably? (which I think we're within here, just curious what are the guidelines here)

Reply:
I think the short description is on the main blog page post card and the long description is at the top when you click into it.  Not sure which works with SEO 🤷🏻‍♀️ @rogermparent, can you help us with SEO and how we might be thinking about this in this respect?

Reply:
Let me know if you think this still needs further extending @jorgeorpinel !

Reply:
> the short description is on the main blog page post card and the long description is at the top when you click into it.

Yep! Also, all the SEO meta tags use the short description. As far as the content, the only big thing to consider when making a description from this standpoint is to prioritize making a good description that draws attention over stuffing it with keywords (though some keywords in the right place can draw attention!), as Google doesn't consider meta description keywords in its ranking algorithm.

![seo-screenshot](https://user-images.githubusercontent.com/9111807/115066881-206e3380-9ebe-11eb-9044-4a795d9f8a80.png)

Reply:
> the short description is on the main blog page post card and the long description is at the top when you click into it

Ah yes now that the PR is deploying I can confirm that. Thanks for the reminder! 

> Not sure which works with SEO 🤷🏻‍♀️ 

Yep. It looks like the short one will go into social media previous (good to know). As for Google search results, they do whatever they want anyway — sometimes they use the suggested title/description, sometimes they build their own.

BTW (since we're tagging @rogermparent 😬) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:

![image](https://user-images.githubusercontent.com/1477535/115067099-0d576580-9eb6-11eb-85ff-ca2bee556a43.png)

Reply:
p.s. just noticed your answer after posting mine, @rogermparent.

Reply:
> making a good description that draws attention over stuffing it with keywords

As long as they match the content yes, otherwise the algo may even penalize ranking for trying to mislead it 😋 or so I've heard.

Reply:
> BTW (since we're tagging @rogermparent 😬) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:

I agree, there's definitely space for it, I can change.  @rogermparent, on the short description for SEO purposes are there length guidelines?  What = short?

Reply:
> What = short?

Probably whatever fits comfortably in our blog cards will roughly work for social media previews too:

![image](https://user-images.githubusercontent.com/1477535/115068620-1a755400-9eb8-11eb-9d52-aa59b55dc7fd.png)

Reply:
> BTW (since we're tagging @rogermparent grimacing) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:
> 
> ![image](https://user-images.githubusercontent.com/1477535/115067099-0d576580-9eb6-11eb-85ff-ca2bee556a43.png)

That's a good idea, we should be able to fetch the long description of the first article in the query and show it on the first card of these index pages. If nothing else, we can make a ticket for the future.

Reply:
https://github.com/iterative/dvc.org/issues/2386